tests/scenarios/test_functionality_nimbus.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pytest_bdd import given, scenario, then
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+@scenario(
+    "../features/generic_functionality.feature",
+    "The browser's URL bar will navigate to the supplied URL",
+)
+def test_browser_navigates_to_url_correctly():
+    pass
+
+
+@given("Firefox is launched enrolled in an Experiment")
+def selenium(selenium):
+    return selenium
+
+
+@then("Firefox should still accept a URL into the search bar")
+def navigate_to_url(selenium):
+    with selenium.context(selenium.CONTEXT_CHROME):
+        el = selenium.find_element(By.CSS_SELECTOR, "#urlbar-input")
+        el.click()
+        el.send_keys("http://www.allizom.org")
```

Comment:
Not sure if you want to use httpS to avoid one of the redirects ("http:" => "https:"; with the second being the "/en-US/" path).
```suggestion
        el.send_keys("https://www.allizom.org")
```

Although possibly quite moot if this doesn't auto-wait for the URLs to do redirection and resolve.